node/update

```diff
@@ -14,11 +14,12 @@ echo "using IP $IP..."
 
 $DIR/stop $IP
 
+# The -o for the unzip command overrides if there's any existing file
 $DIR/remote $IP "\
     echo 'Downloading updated src' && \
-    curl https://s3.amazonaws.com/tp-arqui-node-app-src/src.zip > src.zip && \
+    curl https://s3.amazonaws.com/$(cat ../source_location)/src.zip > src.zip && \
```

Comment:
Cambiá porfa el `../source_location` a algo tipo `$DIR/../source_location` para que no dependa de dónde se ubica la terminal, así se puede correr parado desde donde uno quiera (esa variable se setea como dice [en la segunda respuesta de este post](https://stackoverflow.com/questions/421772/how-can-a-bash-script-know-the-directory-it-is-installed-in-when-it-is-sourced-w).